services/pull/merge_squash.go

```diff
@@ -63,8 +64,9 @@ func doMergeStyleSquash(ctx *mergeContext, message string) error {
 		return err
 	}
 
-	if setting.Repository.PullRequest.AddCoCommitterTrailers && ctx.committer.String() != sig.String() {
-		// add trailer
+	if setting.Repository.PullRequest.AddCoCommitterTrailers &&
+		strings.EqualFold(ctx.committer.Email, sig.Email) {
+		// add trailer if email of user and email of committer don't match
```

Comment:
`don't match`, do you mean `!` `strings.EqualFold` ?

Reply:
You're right. If I was under the impression that this still worked even if it didn't when I tested it, I seriously need to re-examine what I just did.